web/modules/custom/collection/collection.behat.inc

```diff
@@ -83,6 +92,7 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
       'uri' => 'id',
       'name' => 'label',
       'author' => 'author',
+      'description' => 'field_ar_description',
```

Comment:
'ar' stands for Asset Repository which is another RDF enitty. We should change this but this is the name of the description field right now.